applications/ecallmgr/priv/mod_kazoo/events/PLAYBACK_START.xml

```diff
@@ -0,0 +1,5 @@
+<event name="PLAYBACK_START">
+        <fields verbose="true">
```

Comment:
please don't use this ` <fields verbose="true">` , it will get all headers

Reply:
Waiting on resolution of PLAYBACK_START discussion: https://github.com/2600hz/kazoo/pull/6548#pullrequestreview-418419998